seqqc/README.md

```diff
@@ -0,0 +1,39 @@
+# seqqc
```

Comment:
I'd add some details about the samplesheet file and the file paths used in that file. Mainly, currently those file paths are relative and we should note that if you download all of these to test a workflow or smth that you should update them to the full path. 

(like if you download these as they are to a `test/` directory in your seqqc pipeline and call the pipeline from the root of the repo, these files will not work)

Reply:
Your totally right, woops! I meant to makes those the url paths that they’ll be on the main branch, I’ll update them. Thank you!